serialize.go

```diff
@@ -514,6 +514,23 @@ func DeserializeQueryAndArray(context *Context, buffer *Buffer, serializationTyp
 	return array, query, nil
 }
 
+// HandleLoadArraySchemaRequest Pass the array and serialized request to core which returns the serialized response.
```

Comment:
Could this be made more descriptive? Most functions in this package pass something to core, so that doesn’t really illuminate much.

Reply:
Done, thanks!